src/styles/styles.js

```diff
@@ -10,6 +10,25 @@ const globalStyles = StyleSheet.create({
   mapContainer: {
     flex: 1,
   },
+  map: {
+    ...StyleSheet.absoluteFillObject,
+  },
+  // For text input
+  inputStyle: {
+    flex: 1,
+    fontSize: 18,
+  },
+  // Container/styling for text input above
+  backgroundStyle: {
```

Comment:
Global styles should probably only be used for elements that are styled globally. If it's only for a certain component, you should just add the styles within that component file.

Reply:
Also, if this will be used globally for more than one component, try a more specific name. Something like `textInputContainer`.